tesseract_core/sdk/engine.py

```diff
@@ -408,7 +408,12 @@ def build_tesseract(
 
     image_name = config.name
     if image_tag:
-        image_name += f":{image_tag}"
+        tags = [f"{image_name}:{image_tag}"]
```

Comment:
Wondering if we should retire this. If not, then also appending `latest` here may be less surprising.

Reply:
I'm for leaving as it is; it also keeps behaviour as it is right now (i.e., specifying a tag does not also tag the resulting tesseract as `latest`)... but no strong opinion

Reply:
You're right, I thought the Docker default was to *always* add a `latest` tag but apparently that's not the case.